astroplan/core.py

```diff
@@ -37,70 +54,94 @@ def __init__(self, name, longitude, latitude, elevation, pressure,
         name : str
             A short name for the telescope, observatory or location.
 
-        longitude : str or `~astropy.units.quantity`?
-            The longitude of the observing location.
+        pressure : `~astropy.units.Quantity`
+            The ambient pressure.
+
+        location : `~astropy.coordinates.EarthLocation`
+            The location (latitude, longitude, elevation) of the observatory.
+
+        longitude : str or `~astropy.units.quantity`
```

Comment:
`quantity` needs to be `Quantity` or the Sphinx link won't work.
You probably know this, but just in case ... the commands to check the docs locally are:
```
python setup.py build_sphinx
open _docs/_build/html/index.html
```